crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -637,6 +638,24 @@ impl IndexerStore for PgIndexerStore {
         Ok(total_addresses as u64)
     }
 
+    fn get_total_objects(&self) -> Result<u64, IndexerError> {
+        let total_objects = read_only!(&self.cp, |conn| {
+            objects::table
+                .select(count(objects::object_id))
```

Comment:
This works for now but count queries can get pretty slow and expensive depending on data size and db config. long term it might be beneficial to pre-compute these somewhere.

Reply:
agreed, currently we have partitioned objects table, so the count will run a bit of map-reduce out there, the current setup is one partition per epoch, so presumably should be fine.
but yes, cache and pre-compute are easy to add, I can quickly do that if needed

Reply:
oh sry I lied, the objects table are the updated one, so it is NOT partitioned. I will pre-computes then.